/*
try {
    alert("Start of try runs");     // (1) <--

    // lalala;     // 아무의미 없음(에러발생)
    utfdujuce

  alert('End of try runs');   // (2) <--

} catch(err) {

  // alert('Catch is ignored, because there are no errors'); // (3)
  alert(`${err.name}, ${err.message}`);

}
*/
